test/index.js

```diff
@@ -28,7 +27,10 @@ async function assertThrowsAsync (fn, regExp) {
 // 1. run a docker container
 // docker run -i -p 9200:9200 --name my_elastic -p 9300:9300 -e "discovery.type=single-node" elasticsearch:7.2.0
 describe('Test stream events', () => {
+  let es
+
   beforeEach(async () => {
+    es = await elastic(`http://${ES_ENDPOINT}`, true)
```

Comment:
Not sure why you moved this here? I guess this line is responsible for establishing connection, so maybe it would be a good idea to do it only once.

Reply:
I would move `http://` to constant ES_ENDPOINT.